docs/source/reference/image_classification.rst

```diff
@@ -55,6 +55,17 @@ Here's the full example:
     :language: python
     :lines: 14-
 
```

Comment:
I think we could have a new section for this. Don't mind what you call it, `Custom Transformations` or something like that?

Reply:
Yes, `Custom Transformations` seems good